src/pybind/mgr/test_orchestrator/module.py

```diff
@@ -3,6 +3,8 @@
 import threading
 import functools
 import uuid
+import psutil
```

Comment:
ImportError: No module named psutil

Reply:
This does add a new python module dependency -- not sure how best to handle that in automated build systems. Should I add a package dependency to the ceph-mgr specfile?

Reply:
the thing is, it is only meant for qa and developer systems. I'm not sure, if we should add this to the general spec file at all.

Reply:
Probably the right thing to do is to move all of the orchestrator backends into separate packages. Then we could just add this as a dependency to ceph-mgr-test-orchestrator package (or whatever).

Reply:
@dmick any thoughts on this?

Reply:
well, we are currently distributing test_orchestrator.  That may be a mistake; it's a loose end I haven't chased yet.  It's possible it should just not be installed or packaged at all.

Reply:
> the thing is, it is only meant for qa and developer systems. I'm not sure, if we should add this to the general spec file at all.

`install-deps.sh` (used on developer systems, not sure about qa systems) will install all the buildrequires from the rpm spec and/or debian control files, plus a handful of extra xml-related packages. If this dependency is absolutely only necessary on developer systems, it could go in the `install-deps.sh` script, otherwise it belongs in the spec file / control file.

> as for psutil: sigh, do we _have_ to have another external python dependency?

psutil at least should be quite widely available..

Reply:
I don't know. continuing with `ps´ won't be a good idea in the long run. adding this as dependency without it being used in a production environment is also not so great.